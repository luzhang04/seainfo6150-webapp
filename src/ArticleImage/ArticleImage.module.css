.container {
  border-radius: 50%;
  width: 150px;
  height: 100px;
  border: 10px solid #c00;
  margin-bottom: 1.5rem;
}
