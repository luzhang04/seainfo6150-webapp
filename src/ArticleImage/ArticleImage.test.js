import React from "react";
import ArticleImage from "./ArticleImage";


describe("ArticleImage tests", () => {
  it("renders correctly", () => {
    const article1 = {

            title : "title",
            url: "www.google.com"

    }
    const { container } = render(<ArticleImage title={article1.title} url={article1.url}/>);
    expect(container).toMatchSnapshot();
  });
});
