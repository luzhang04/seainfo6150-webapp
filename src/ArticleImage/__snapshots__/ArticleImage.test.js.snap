// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleImage tests renders correctly 1`] = `
<div>
  <img
    alt="title"
    class="image"
    src="www.google.com"
  />
</div>
`;
